uint8 WHEEL_FRONT_LEFT=1
uint8 WHEEL_FRONT_RIGHT=2
uint8 WHEEL_REAR_LEFT=3
uint8 WHEEL_REAR_RIGHT=4

# We use these numbers so that the same ID can be used for the velocity and steering
uint8 WHEEL_MIDDLE_LEFT=5
uint8 WHEEL_MIDDLE_RIGHT=6


float32[6] velocities_rad_per_sec
float32[4] steering_rad
